chatbot/workflows/update_vectordb.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
Please give a better name to this file. I think you can call it as prepare-data or any other proper name?

Reply:
Good suggestion, I want to rename it as "md_link_updater.py", as it just traverses the folder and updates the links of md files. And it is triggered by the workflow in .github.